src/h5web/visualizations/nexus/models.ts

```diff
@@ -15,8 +15,6 @@ export enum NxInterpretation {
   Image = 'image',
 }
 
-export const NX_INTERPRETATIONS = Object.values<string>(NxInterpretation);
```

Comment:
I changed my mind on this: keeping this as a constant outside of `isNxInterpretation` is an over-optimisation.